getting-started/pull-request-lifecycle.rst

```diff
@@ -180,7 +187,14 @@ Making Good PRs
 When creating a pull request for submission, there are several things that you
 should do to help ensure that your pull request is accepted.
 
-First, make sure to follow Python's style guidelines. For Python code you
+First, make your change against the right version of Python. Even if you're
+fixing a bug, your fix needs to go to the ``main`` branch first. After the
+fix is merged, it will be :ref:`ported back <branch-merge>` to older
+maintenance releases as well. That way we ensure all affected versions are
+handled. Unless you're sure that your change does not apply to ``main``,
+don't base your fixes on old versions of Python.
+
+Second, make sure to follow Python's style guidelines. For Python code you
```

Comment:
Sidenote: To avoid noisy, tedious and easy to mess up manual renumbering like this, a long list of prose-numbered paragraphs, and where there isn't an inherent order, it would be better to reformat this as an unordered (bulleted) list instead (or at least use an auto-numbered ordered list). But perhaps out of scope here.

Reply:
I remember we already deferred doing this once to a separate PR so I'll just do it here then.